### ISSUE TITLE
A11y_Visual Studio Code Client_Sep2021_Debug Adapter Protocol_Debug Adapter Protocol specification_Zoom: On 400% zoom, Right side links will get invisible.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/debug-adapter-protocol#209]
“C[heck out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTag**:#A11yTCS;#DesktopWeb;#Win10;#WCAG1.4.10;#FTP;#A11yMAS;#A11yWCAG2.1#ChromiumEdge;#A11ySev1;#P2_Visual Studio Code Clien_Sep2021;#Zoom;#Visual Studio Code Client;#P2_WCP;

## **Environment Details:**

Application Name: Visual Studio Code Client
#URL: https://microsoft.github.io/debug-adapter-protocol
Browser Versions: Microsoft Edge Version 93.0.961.38 (Official build) (64-bit)
Windows Version: Windows10

**Example Page URL**: https://microsoft.github.io/debug-adapter-protocol/specification

## **Prerequisite:**
Apply 400% zoom and Set Resolution as 1280x1024
## **Repro Steps:**

1. Open the URL: https://microsoft.github.io/debug-adapter-protocol/specification
2. Debug Adapter Protocol specification screen will get open..
3. Zoom screen upto 400% and check that right side navigation controls are visible or not..

## **Actual:**

On 400% zoom, Right side links will get invisible.

**Same issue is also observed with Scenario "Language Server Protocol Specification" .**

## **Expected:**

All the controls should be visible and accessible by keyboard on 400% zoom.

## **User Impact:**

Users will not be able to see all the content present on screen in 400% zoom.

## **Attachment:**
![#209_Narrator](https://user-images.githubusercontent.com/60422426/133751426-cf7b0d69-951b-481f-8920-ec9f41dee798.gif)

**Comment by sana-ajani** [comment]
I am able to see all the content on 400% zoom. The code blocks on this page the debug-adapter-protocol/specification page have horizontal scroll. The right side links fall into a hamburger menu that can be accessed by tabbing. I don't see this as a bug.

**Comment by NiharikaTipirishetty** [comment]
#P2_debugadapterprotocol_Web_Sep2021; #P2_languageserverprotocol_Web_Sep2021;

**Comment by sripradhan** [comment]
Regressed in env: https://microsoft.github.io/debug-adapter-protocol/specification
Issue is fixed. Added attachment for reference.

![Screenshot (168)](https://user-images.githubusercontent.com/87061368/178192690-1ddeafa6-faad-4660-944a-603c302ed14e.png)

**Comment by prabhudevu** [comment]
#Closed;
